stdlib/public/core/UnsafeBufferPointer.swift.gyb

```diff
@@ -328,7 +328,22 @@ public struct Unsafe${Mutable}BufferPointer<Element>
     self.init(start: slice.base.baseAddress! + slice.startIndex,
       count: slice.count)
   }
-%  end # !mutable
+
+  @_inlineable
+  public init(_ other: UnsafeMutableBufferPointer<Element>) {
```

Comment:
Please make sure any new public APIs have documentation before this gets merged—thanks!